client/dist/fonts/silverstripe.svg

```diff
@@ -106,6 +105,12 @@
 <glyph glyph-name="lock" unicode="&#57348;" d="M404 321c7 0 14-4 21-10 7-7 10-14 10-21 0 0 0-200 0-200 0-16-8-28-24-34 0 0-31-9-31-9-14-5-31-8-49-8 0 0-149 0-149 0-19 0-36 3-50 8 0 0-31 9-31 9-16 6-24 18-24 34 0 0 0 200 0 200 0 7 2 14 7 21 6 6 12 10 18 10 0 0 52 0 52 0 0 0 0 35 0 35 0 38 8 67 26 87 17 21 43 31 76 31 33 0 59-10 76-31 18-20 26-49 26-87 0 0 0-35 0-35 0 0 46 0 46 0m-199 46c0 0 0-46 0-46 0 0 102 0 102 0 0 0 0 46 0 46 0 17-4 31-14 41-9 10-21 15-37 15-16 0-28-5-37-15-10-10-14-24-14-41"/>
 <glyph glyph-name="cross-mark" unicode="&#57349;" d="M414 159l-97 97 97 97c17 17 17 44 0 61-17 17-44 17-61 0l-97-97-97 97c-17 17-44 17-61 0-17-17-17-44 0-61l97-97-97-97c-17-17-17-44 0-61 8-8 19-13 30-13 11 0 23 5 31 13l97 97 97-97c9-8 20-13 31-13 11 0 22 5 30 13 17 17 17 44 0 61z"/>
 <glyph glyph-name="folder-add" unicode="&#57350;" d="M384 213l-43 0 0-42-42 0 0 42-43 0 0 43 43 0 0 43 42 0 0-43 43 0z m-139 150l-41 64-146 0c-9 0-15-7-15-15l0-289c0-9 7-16 15-16 66 0 307 0 373 0 10 0 17 7 17 16l0 223c0 9-7 17-17 17-48-1-186 0-186 0z"/>
-<glyph glyph-name="folder-move" unicode="&#57351;" d="M299 170l0 43-64 0 0 43 64 0 0 42 85-63z m-95 257l-146 0c-9 0-15-7-15-15l0-289c0-9 7-16 15-16 66 0 307 0 373 0 10 0 17 7 17 16l0 223c0 9-7 17-17 17-48-1-186 0-186 0z"/>
```

Comment:
folder-move was renamed to move-folder; Should we revert that to prevent breakages?

Reply:
fixed with the new commit coming up